src/lib/project/twitter.ex

```diff
@@ -98,4 +99,51 @@ defmodule Project.Twitter do
   def delete_post(%Post{} = post) do
     Repo.delete(post)
   end
+
+  @doc """
+  あるユーザーのお気に入り登録一覧のリストを返す。
+  """
+  @spec list_favorite_posts(integer) :: list(Favorite.t())
+  def list_favorite_posts(user_id) do
+    query = from f in Favorite,
+      where: f.user_id == ^user_id
+    Repo.all(query)
```

Comment:
```suggestion
    Favorite
    |> where([f], f.user_id == ^user_id)
    |> Repo.all()
```
function styleで書きましょう！

Reply:
他の箇所も同様に修正して下さい